### DISCUSSION TITLE
IPAdapter for flux?

### DISCUSSION THREAD

**Discussion by cchance27** [cubiq/ComfyUI_IPAdapter_plus#672] [General]
Just wondering since it seems full fine tunes may be a while off, and loras are.. a LOT for flux, was wondering if anyone knows if maybe we'll see IPAdapter being worked on for flux dev or schnell?

**Comment by cubiq** [comment]
@xiaohu2015 any news on SD3 or Flux models? I tried to contact you via email but it says I'm not authorized to write you :smile:

**Comment by xiaohu2015** [comment]
hi, I have left Tencent.

**Reply by cubiq** [reply]
good luck for your new endeavors!

**Reply by xiaohu2015** [reply]
If you have some questions, you can email me xiaohuzc@gmail.com
